testcases/kernel/syscalls/sync_file_range/sync_file_range02.c

```diff
@@ -124,10 +125,6 @@ static void setup(void)
 static struct tst_test test = {
 	.tcnt = ARRAY_SIZE(testcases),
 	.needs_root = 1,
-	.mount_device = 1,
```

Comment:
Please add below config parameters to tst_test test structure initialization in fdatasync03.c  file.
.mount_device = 1,
.all_filesystems = 1,
.mntpoint = MNTPOINT,

By enabling above options, test framework will mount the file system at the time of test initialization time.

Reply:
@shaikshavali1 we have already /data and /dev/vdb mounted. Why are we mounting again? Do we really need to mount again?